docker/user_demos.sh

```diff
@@ -22,6 +22,15 @@ if [ "$PYTHON" = "miniconda" ]; then
       # delete GPU deps
       sed -r -e '/^\s*- (astra-toolbox|tigre).*/d' -e '/^\s*- \S+.*#.*GPU/d' environment.yml > environment-sirf.yml
     fi
+    # do not install CIL from conda if BUILD_CIL is set
+if test "${BUILD_CIL:-OFF}" != "OFF"; then
+      # delete CIL package from the environment file
+      echo "Deleting CIL from the environment file BUILD_CIL is set to >${BUILD_CIL}<"
+      sed -r -i -e '/^\s*- (cil).*/d' environment-sirf.yml
```

Comment:
Isn't this better now @paskino ?
```suggestion
      sed -r -i -e '/# cil/d' environment-sirf.yml
```

Reply:
Oh actually it applies to https://github.com/SyneRBI/SIRF-Exercises/blob/master/environment.yml, in wguch case there's no `cil-data`, right?